src/OwnMap.java

```diff
@@ -0,0 +1,11 @@
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class OwnMap<K, V> extends HashMap<K, V> {
```

Comment:
use lambda

Reply:
do we need this class if we override this method in class MailService?